src/site/_collections/pages.js

```diff
@@ -116,43 +108,25 @@ module.exports = (collections) => {
     }
 
     const defaultUrl = getDefaultUrl(item.url);
+    const paginated = !!item.data.pagination;
+    const content =
+      paginated && !!item.data?.renderData?.description
+        ? item.data.renderData.description
```

Comment:
Chatted with Michael — We use renderData because we need to use the i18n filter in the frontmatter for these pages. We should refactor at least the author and tag pages to use eleventyComputed.